gulp/tasks/copy.js

```diff
@@ -1,100 +1,108 @@
-const gulp = require( 'gulp' );
-const gulpChanged = require( 'gulp-changed' );
-const handleErrors = require( '../utils/handle-errors' );
-const paths = require( '../../config/environment' ).paths;
-const path = require('path');
+const gulp = require("gulp");
+const gulpChanged = require("gulp-changed");
+const handleErrors = require("../utils/handle-errors");
+const paths = require("../../config/environment").paths;
+const path = require("path");
```

Comment:
Mike already touched on this in his `gulpfile.js` comment, but we don't want to apply large scale formatting changes (adding/removing spaces, changing `"` to `'`, etc) to any Refresh files.  

I would recommend disabling the `Format on Save` option in your code editor to help avoid this.